src/main/java/com/gmail/goosius/siegewar/SiegeWar.java

```diff
@@ -90,8 +91,9 @@ public void onEnable() {
 		registerPlayerCommands();
 		registerListeners();
 		checkIntegrations();
-		migrateTownOccupationData();
+		migrateTownOccupationData(); //Keep this line before deleteLegacyMetadata
 		deleteLegacyMetaData();
+		migrateTownNeutralityData();

```

Comment:
Now that there's three things going on, this should probably get put into its own method where comments can be added to make sure things stay in the right order.

I would probably also kick it over to a new LegacyMetaDataHandler class.

Reply:
Haha, very good. Before I read this, this morning, I did just that.